skaffold.yaml

```diff
@@ -39,7 +39,7 @@ deploy:
         createNamespace: true
         remoteChart: redis
         repo: https://charts.bitnami.com/bitnami
-        version: 17.6.0
+        version: 18.2.1
```

Comment:
There's a newer version `18.2.2` now, should we use it?

Reply:
Same as above